src/events/pg.js

```diff
@@ -0,0 +1,122 @@
+const uuid4 = require('uuid4');
+const shimmer = require('shimmer');
+// eslint-disable-next-line no-unresolved
+const pg = require('pg');
+const sqlParser = require('node-sqlparser');
+const utils = require('../utils.js');
+const tracer = require('../tracer.js');
+const serverlessEvent = require('../proto/event_pb.js');
+const eventInterface = require('../event.js');
+const errorCode = require('../proto/error_code_pb.js');
+
+/**
+ * Wraps the pg's module request function with tracing
+ * @param {Function} wrappedFunction The pg's module
+ * @returns {Function} The wrapped function
+ */
+function pgClientWrapper(wrappedFunction) {
+    return function internalpgClientWrapper(queryString, arg1, arg2) {
+        // eslint-disable-next-line no-var
+        var patchedCallback;
+
+        const callback = (arg2 === undefined) ? arg1 : arg2;
+        const params = (arg1 instanceof Function) ? [] : arg1;
+        let sqlObj = {};
+
+        try {
+            sqlObj = sqlParser.parse(queryString);
+        } catch (error) {
+            tracer.addException(error);
```

Comment:
not sure if we should add a tracer exception here (this will explode for every time the user runs an invalid query). lets just make sure we always add the queryString to the event, even if the parsing fails